instrumentation/micrometer/micrometer-1.5/library/src/main/java/io/opentelemetry/instrumentation/micrometer/v1_5/Bridging.java

```diff
@@ -36,9 +37,13 @@ static String name(Meter.Id id, NamingConvention namingConvention) {
     return namingConvention.name(id.getName(), id.getType(), id.getBaseUnit());
   }
 
-  static String description(Meter.Id id) {
-    String description = id.getDescription();
-    return description == null ? "" : description;
+  static String description(String name, Meter.Id id) {
```

Comment:
Could you add a TODO here to remove that once SDK stops throwing exceptions?
```suggestion
  // TODO: remove the cache usage once the SDK is able to handle different descriptions
  static String description(String name, Meter.Id id) {
```